src/solve.jl

```diff
@@ -1,15 +1,10 @@
 function SciMLBase.__solve(prob::Union{SciMLBase.AbstractSteadyStateProblem,
                                        SciMLBase.AbstractNonlinearProblem},
                            alg::algType,
-                           reltol = 1e-3,
+                           args...;
                            abstol = 1e-6,
```

Comment:
Can we forward reltol to xtol for now?

Reply:
This is a bad idea because it isn't a `reltol` and no other nonlinear solvers pretend that `reltol` means `xtol`

Reply:
Sure. That's fine.